mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -127,10 +128,16 @@ final class OneVsRestModel private[ml] (
       predictions.maxBy(_._2)._1.toDouble
     }
 
+    // output the index of the classifier with highest confidence as prediction
+    val probabilityUDF = udf { (predictions: Map[Int, Double]) =>
```

Comment:
Sorry for the delayed response. I've made some small changes as per your recommendations. WRT this point, I'm not sure I follow. As far as I understand, I need to return the highest confidence factor(which will be Double) corresponding to the model used with OVR.